banzai/images.py

```diff
@@ -40,13 +40,15 @@ def __init__(self, pipeline_context, filename=None, data=None, header=None,
 
         self.request_number = header.get('REQNUM')
 
-        self.site = header.get('SITEID')
-        self.instrument = header.get('INSTRUME')
-        if self.site is not None and self.instrument is not None:
-            self.telescope_id = dbs.get_telescope_id(self.site, self.instrument,
-                                                     db_address=pipeline_context.db_address)
-        else:
-            self.telescope_id = None
+        if len(self.header) > 0:
```

Comment:
Is this line to correctly pass FakeImages?

Reply:
Fixed.